minimal-notebook/ipython_notebook_config.py

```diff
@@ -4,9 +4,9 @@
 PEM_FILE = os.path.join(os.path.dirname(__file__), 'security/notebook.pem')
 
 c = get_config()
-c.NotebookApp.ip = '*'
+c.NotebookApp.ip = os.getenv('INTERFACE', '*')
```

Comment:
I like to use `os.getenv('Key', '') or 'DEFAULT'` to handle the case of `KEY=`, which should be treated the same as KEY being undefined.